examples/step-3/doc/results.dox

```diff
@@ -224,3 +224,213 @@ suggestions:
   Again, the difference between two adjacent values goes down by about a
   factor of four, indicating convergence as ${\cal O}(h^2)$.
 </ul>
+<h4>Using the %HDF5 Library to output the solution and additional data</h4>
+%HDF5 is a commonly used format that can be read by many scripting languages (e.g. R or Python). Here are some 
+insights on what is possible.
+<h5> Changing the output from .gnuplot to .h5:</h5>
```

Comment:
I know I recommended this but it turns out that `<h5>` ends up being smaller than the surrounding text. The easiest way to fix this would be to change `Possibilities for extensions` to `<h2>` and similarly decrement the following sections so that the highest heading level is `<h5>`.